src/cli/Cli.spec.ts

```diff
@@ -400,6 +463,20 @@ describe('Cli', () => {
       }, e => done(e));
   });
 
+  it(`passes options validation if the command specified has boolean arguments of several truthy/falsy values`, (done) => {
+    cli
+      .execute(rootFolder, ['cli', 'mock', 'boolean', 'curation', '--parameterA', '1', '-b', '0', '-c', 'true', '-d', 'false', '-e', 'yes', '-f', 'no', '-g', 'on', '--parameterH', 'off'])
```

Comment:
The problem with putting all these value in one test is, that if one of them fails, we have no idea which one, without debugging the test. It would be easier to define a separate test condition for each one, eg. `passes validation option for numeric booleans (0, 1)`, `passes validation for on, off as booleans` etc. That way, when something fails, we know exactly what's wrong.